target/linux/ramips/dts/mt7621_bolt_arion.dts

```diff
@@ -100,9 +100,24 @@
 			};
 
 			factory: partition@40000 {
+				compatible = "nvmem-cells";
 				label = "factory";
 				reg = <0x40000 0x10000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
 				read-only;
+
+				eeprom_factory_0: eeprom@0 {
+					reg = <0x0 0x400>;
+				};
+
+				eeprom_factory_8000: eeprom@8000 {
+					reg = <0x8000 0x200>;
+				};
+
+				macaddr_factory_4000: macaddr@4000 {
+					reg = <0x28 0x6>;
```

Comment:
Idea for this fix was to change this in a previous commit and then make the convert commit just move the node inside.
Since you have to touch this pr again, can you do this change?

Reply:
You mean nesting nvmem stuff into partition nodes in the previous commit? It should be fine now.
https://github.com/openwrt/openwrt/pull/13587/commits/6f52c1429a29e0594df10d7df223366a41e29ff9

Reply:
No I mean that before all these commit, you create a single simple commit that just fix the wrong node name and tag.

And then in the conversion you move the node in the partition nodes as you did in the linked commit.

Reply:
Oh, it seems that you didn't notice this small commit before, I've already done it. https://github.com/openwrt/openwrt/commit/001f05b220e3d2a0419026ce50332c8b9a33cc69
I'll force push it. Thanks again. It's really a hard job to review these changes.

Reply:
Oh it was there and I didn't notice! Sorry!!